main.scss

```diff
@@ -20,3 +20,7 @@
 @import "src/scss/field-errors";
 @import "src/scss/field-valid";
 @import "src/scss/global-and-section-messages";
+
+// Set silent mode back on again to avoid it being output twice
+$o-forms-is-silent: true;
+$o-ft-forms-is-silent: true;
```

Comment:
Cool with me. The v1 sticks to the spec.

We should probably keep that technique in o-grid for now (and violate to the spec), because it's being required by a lot of different modules and I think we'd see an explosion in file size (38KB each time).


For the record here's the bit in o-grid that prevents it from being output multiple times:

```scss
@if $o-grid-is-silent == false {
	@include oGridSurfaceCurrentLayout;
	@include oGridGenerate;

	// Turn silent mode back on to avoid outputting the grid twice
	$o-grid-is-silent: true;
}
```